docs/blueprint.md

```diff
@@ -54,6 +55,11 @@ git clone https://github.com/awslabs/data-on-eks.git
 ```bash
 cd data-on-eks/analytics/terraform/clickhouse-eks
 chmod +x install.sh
+
```

Comment:
I would remove this from blueprint, and actually I am not sure if we should include this in readme as well. We are setting aws cli as requirement and people should take care about setting it up properly with the auth strategy they want.

Reply:
probably yes. I would still appreciate it as a reminder, if I don't use is regularly (creds for was cli).

install.sh - probably. I thought it was not finished yet, so just not touching it.